api/queries_pr.go

```diff
@@ -508,34 +503,18 @@ func PullRequestStatus(client *Client, repo ghrepo.Interface, options StatusOpti
 	return &payload, nil
 }
 
-func pullRequestFragment(httpClient *http.Client, hostname string) (string, error) {
-	cachedClient := NewCachedClient(httpClient, time.Hour*24)
-	prFeatures, err := determinePullRequestFeatures(cachedClient, hostname)
-	if err != nil {
-		return "", err
-	}
-
+func pullRequestFragment() string {
```

Comment:
I updated this function to no longer use feature detection. All the features detected here have exist on GHES 3.0 or earlier. GHES 3.0 was discontinued on 2022-02-16, so we no longer need to support feature detection here and just assume these fields exist in all cases.